### PR TITLE
Adds status() callout.

### PR DESCRIPTION
**Pull request by johnnymatthews**
Tells users that `status()` exists, and that they can use it.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!


### `docs/how-tos/query.md`:
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check for null...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```

File: docs/README.md [hunk 1]
```diff
@@ -144,6 +144,10 @@ Now that you've got your script ready to go, you just need to run it in your ter
 
 1. **Make a note of the CID, which looks like `bafyb...`.** You'll need it in order to get your file.
 
+:::tip Get the status of your upload
+It's possible to get the status of your file upload by using the `status()` method within the JavaScript client library. Checkout the [Query how-to section for more information →](./how-tos/query/)
```
Comment (jessicaschilling) [inline docs/README.md:148]:
```suggestion
It's possible to get the status of your file upload by using the `status()` method within the JavaScript client library. Check out the [Query how-to guide](./how-tos/query.md) for more information.
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!


### `docs/how-tos/query.md`:
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check for null...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```

**Review by jessicaschilling — APPROVED** [review]

**Comment by terichadbourne** [comment]
@johnnymatthews If you said "status of your upload" to me without context, I would expect to hear something like 32% uploaded. When I go to the how to it doesn't seem like any of the data returned is about how uploaded the file is. Is there another way to describe the kind of info it helps you with? 

Is it intentional that you linked to the start of all of the how tos instead of linking directly to the status section?

File: docs/README.md [hunk 2]
```diff
@@ -144,6 +144,10 @@ Now that you've got your script ready to go, you just need to run it in your ter
 
 1. **Make a note of the CID, which looks like `bafyb...`.** You'll need it in order to get your file.
 
+:::tip Get the status of your upload
+It's possible to get the status of your file upload by using the `status()` method within the JavaScript client library. Check out the [Query how-to guide](./how-tos/query.md) for more information.
```
Comment (jessicaschilling) [inline docs/README.md:148]:
```suggestion
It's possible to get a variety of details about your upload, including its CID, upload date, size on the network, and info on IPFS pinning and Filecoin storage deals, by using the `status()` method within the JavaScript client library. Check out the [Query how-to guide](./how-tos/query.md#querying-for-status-information) for more information.
```
@terichadbourne Does this take care of your concern?

Comment (johnnymatthews) [reply docs/README.md:148]: I'm happy with this if you are @terichadbourne.

**Review by terichadbourne — APPROVED** [review]
Good to go with the change (which I just committed). Just waiting on the checks, then can be merged.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!


### `docs/how-tos/query.md`:
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check for null...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...erence.  If you haven't yet explored in depth how to store data using Web3.S...
                                             ^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!


### `docs/how-tos/query.md`:
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check for null...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...erence.  If you haven't yet explored in depth how to store data using Web3.S...
                                             ^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```

**Pull request merged**